files/usr/share/cinnamon/applets/inhibit@cinnamon.org/applet.js

```diff
@@ -416,6 +417,15 @@ class CinnamonInhibitApplet extends Applet.IconApplet {
     on_btn_open_system_notification_settings_clicked() {
         Util.spawnCommandLine("cinnamon-settings notifications");
     }
+    
+    on_applet_added_to_panel() {
+		if (this.inhibitPowerAtStartup) {
+			let id = setTimeout ( () => {
+				this.toggle_inhibit_power();
+				clearTimeout(id);
+			}, 1000); // Let 1000 ms to have this.sessionProxy not null.
+		}
+	}
```

Comment:
I've noticed on this and other pull requests that you're using tabs instead of spaces. The coding guidelines highlight [here](https://linuxmint-developer-guide.readthedocs.io/en/latest/guidelines.html#consistency) and [here](https://linuxmint-developer-guide.readthedocs.io/en/latest/guidelines.html#indentation) about why not to do so.

Reply:
Fixed.